playwright-e2e/tests/dsm/participant-page/participant-page-tag.spec.ts

```diff
@@ -0,0 +1,104 @@
+  test.beforeEach(async ({ page }) => {
+    await login(page);
+  });
+
+  test('Ensure cohort tags update and delete properly for Brain @dsm @dsm-search @functional', async ({ page }) => {
+    await cohortTagTest('Brain', page);
+  });
+
+  test('Ensure cohort tags update and delete properly for PanCan @dsm @dsm-search @functional', async ({ page }) => {
+    await cohortTagTest('PanCan', page);
+  });
+});
+
+async function cohortTagTest(studyName: string, page: Page) {
+  //Get current date and time to ensure tag is different from others
+  const currentdate = new Date();
+  //Go to study
+  await new Select(page, { label: 'Select study' }).selectOption(studyName);
+  await expect(page.locator('h1')).toHaveText('Welcome to the DDP Study Management System');
+  await expect(page.locator('h2')).toHaveText(`You have selected the ${studyName} study.`);
+
+  await study(page).selectOption('Participant List', { waitForNav: true });
+  await expect(page.locator('h1')).toHaveText('Participant List', { timeout: 30 * 1000 });
+
+  const participantListPage = new ParticipantsPage(page);
+  await participantListPage.waitForReady();
+
+  //Filter to only patients with medical records
+  await participantListPage.filterMedicalRecordParticipants();
+  await participantListPage.waitForReady();
+  const table = new Table(page);
+  //Make sure we got at least one result and click the first result.
+  expect(await table.rowLocator().count()).toBeGreaterThanOrEqual(1);
+  const cell = await table.cell(0, 2);
+  await cell.click();
+  await expect(page.locator('h1')).toHaveText('Participant Page', { timeout: 5 * 1000 });
+
+  //Make new tags
+  await page.fill('[placeholder="New tag..."]', `dsm_rc_${currentdate}_${studyName}`);
+  await page.keyboard.press('Enter');
+  //Delete tag
+  await page.locator(`text=dsm_rc_${currentdate}_${studyName}`).click();
+  await page.keyboard.press('Backspace');
+  //Make new tag
+  await page.fill('[placeholder="New tag..."]', `DSM RC ${studyName} ${currentdate}`);
+  await page.keyboard.press('Enter');
+  //Make new tag
+  await page.fill('[placeholder="New tag..."]', `${studyName} ${currentdate}`);
+  await page.keyboard.press('Enter');
+  //Make test note
+  await page.fill('textarea:right-of(:text("Participant Notes"))', `This is a test note - ${currentdate}`);
+  await page.keyboard.press('Tab');
+  //Refresh page
+  await page.reload();
+  await expect(page.locator('h1')).toHaveText('Participant List', { timeout: 30 * 1000 });
+  await participantListPage.waitForReady();
+  await participantListPage.filterMedicalRecordParticipants();
+  await cell.click();
+  await expect(page.locator('h1')).toHaveText('Participant List', { timeout: 30 * 1000 });
+  //Make sure first tag was deleted
+  await expect(page.locator(`text=dsm_rc_${currentdate}_${studyName}`)).toHaveCount(0);
+
+  //Make sure second tag is still there, then delete it
+  await expect(page.locator(`text=DSM RC ${studyName} ${currentdate}`)).toHaveCount(1);
```

Comment:
Here is a xpath to get firsst tag text:
`'//mat-form-field[.//input[@id=//*[@aria-label="Tag selection"]/@data-mat-chip-input]]//*[@role="option"][1]/text()[normalize-space()]'`

<img width="1042" alt="Screenshot 2023-01-18 at 5 06 46 PM" src="https://user-images.githubusercontent.com/35533885/213305834-1ac74f81-f0a4-40e0-b0da-3e3b792335c9.png">